tests/specs/fiber_context.js

```diff
@@ -0,0 +1,73 @@
+describe('Fiber based context', function () {
+  var fiberServer = meteor({fiber: true});
```

Comment:
@arunoda What if we make it configurable? So for example:
```
gagarin --default-flavor fibers
```
will make all your `meteor()` calls to create fiber-based servers by default. To make things more consistent we could also use the same word for options:
```
meteor({ flavor: 'promises' });
meteor({ flavor: 'fibers' });
```
It does not have to be called "flavor". Do you have a better idea for the name?